integration-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+
+echo "Reset customizations"
+rm -f sentry/enhance-image.sh
+rm -f sentry/requirements.txt
+
+echo "Testing initial install"
+./install.sh
+./_integration-test/run.sh
+./_integration-test/ensure-customizations-not-present.sh
```

Comment:
Why is this a separate script vs. called in/under `run.sh`? Same below with `ensure-customizations-work.sh`.

Reply:
Because they are logical opposites.